packages/eslint-config/eslint.config.mjs

```diff
@@ -0,0 +1,114 @@
+import js from '@eslint/js';
+import json from '@eslint/json';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import perfectionist from 'eslint-plugin-perfectionist';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+//import nlDesignSystemJS from './rules/js.mjs';
+//import nlDesignSystemReact from './rules/react.mjs';
+
+export default tseslint.config(
+  {
+    name: 'nl-design-system/global-ignores',
+    ignores: ['**/dist/', '**/build/', '**/coverage/'],
+  },
+  {
+    name: 'nl-design-system/plugins-settings',
+    plugins: { perfectionist, react },
+    settings: {
+      react: {
+        version: 'detect',
+      },
+    },
+  },
+  {
+    name: '@eslint/js/recommended',
+    files: ['**/*.js', '**/*.cjs', '**/*.mjs', '**/*.jsx', '**/*.ts', '**/*.tsx'],
+    ...js.configs.recommended,
+  },
+  {
+    name: 'typescript-eslint/configs/strict',
+    extends: [...tseslint.configs.strict],
+    files: ['**/*.ts', '**/*.mts', '**/*.tsx'],
+  },
+  {
+    name: 'eslint-plugin-perfectionist/recommended-natural',
+    ...perfectionist['recommended-natural'],
+  },
+  {
+    name: 'eslint-plugin-react/recommended',
+    files: ['**/*.jsx', '**/*.tsx'],
+    ...react.configs.flat.recommended,
+  },
+  {
+    name: 'eslint-plugin-react/jsx-runtime',
+    files: ['**/*.jsx', '**/*.tsx'],
+    ...react.configs.flat['jsx-runtime'],
+  },
+  {
+    // Source code that ends up in a browser
+    name: 'nl-design-system/browser',
+    files: ['**/src/*.ts', '**/src/*.tsx'],
+    languageOptions: {
+      globals: { ...globals.browser },
+    },
+  },
+  {
+    // Node.js based scripts and tooling configuration
+    name: 'nl-design-system/node',
+    files: ['**/*.js', '**/*.cjs', '**/*.mjs'],
+    languageOptions: {
+      globals: { ...globals.node },
+    },
+  },
+  {
+    // Tests
+    name: 'nl-design-system/tests',
+    files: ['**/*.test.ts', '**/*.test.tsx'],
+    languageOptions: {
+      globals: { ...globals.jest },
```

Comment:
Volgens mij is deze niet strict noodzakelijk omdat `.tsx` door typescript-eslint gehaald wordt en daarin staat `no-undef` uit waardoor dingen als `describe` en `it` niet opgepikt worden. Maar omdat het TS is wordt het wel door TS zelf geflagged als types ontbreken.

Reply:
expliciete imports FTW
https://github.com/nl-design-system/themes/blob/main/packages/theme-toolkit/src/ExampleTokensCSS.test.ts#L1C1-L2C1

Reply:
weggehaald